collector/src/main/java/io/prometheus/jmx/JmxScraper.java

```diff
@@ -245,13 +242,13 @@ private void processBeanValue(
             String attrDescription,
             Object value) {
         if (value == null) {
-            logScrape(domain + beanProperties + attrName, "null");
+            LOGGER.log(Level.FINE, "%s%s%s scrape: null", domain, beanProperties, attrName);
         } else if (value instanceof Number || value instanceof String || value instanceof Boolean || value instanceof java.util.Date) {
             if (value instanceof java.util.Date) {
                 attrType = "java.lang.Double";
                 value = ((java.util.Date) value).getTime() / 1000.0;
             }
-            logScrape(domain + beanProperties + attrName, value.toString());
+            LOGGER.log(Level.FINE, "%s%s%s scrape: %s", domain, beanProperties, attrName, value);
```

Comment:
I think a static import would make it more readable, like `LOGGER.log(FINE, ...)`.

Reply:
Changed to static import.